src/compiler/proposals-and-stabilization/custom-test-jobs.md

```diff
@@ -0,0 +1,96 @@
+# Adding custom test jobs or adding custom test components to rust-lang/rust CI
+
+## Scope
+
+This policy is applicable to proposals for adding new test jobs or adjusting existing test jobs to build additional custom things that are *not* shipped to the end users, but may still cause [rust-lang/rust] PR CI or Full CI (the "CI") to fail, or produce a failure message, which may impact compiler/library/tools contributors who also use the [rust-lang/rust] CI.
+
+For example, this includes but is not limited to:
+
+- Ecosystem test jobs, such as Rust for Linux or Fuchsia;
+- Custom codegen backend test components, such as `codegen_cranelift` or `codegen_gcc`, which are not part of the "main" distributed toolchain.
```

Comment:
`cg_clif` has been shipping with rustup for quite some time, so I think that's a bit different from `cg_gcc`. If we ship the thing with rustup, I think that it essentially becomes the "main" distributed toolchain.

Reply:
Makes sense to me. I think I will remove cg_clif from this list since it's indeed part of the "main" distributed toolchain. (We're still setting up a ping group for cg_clif in https://github.com/rust-lang/rust/pull/138120, but that's mostly for consultation purposes.)

Reply:
Yeah, these are two separate things (is main toolchain vs has more than ~one person who can fix it :) ).

Reply:
Remove cg_clif from this list.